containers/Makefile

```diff
@@ -152,6 +168,15 @@ build/repositories/concent/: build/package-builder/clone-or-update.sh /var/conce
 build/repositories/golem/: build/package-builder/clone-or-update.sh /var/concent-data/golem
 	build/package-builder/clone-or-update.sh  "golem" "$@"
 
+build/signing-service/signing-service/RELEASE-VERSION: build/signing-service/signing-service/
+	echo -n "$(shell cd build/repositories/concent/; git describe --tags)" > "$@"
+
+build/signing-service/signing-service/: build/repositories/concent/
+	cp --recursive "$<signing_service/." "$@"
```

Comment:
If it will affect how the Makefile works, you can leave it as is. But if not, I'd prefer to change it for the sake of readability. I was very confused when I have seen this for the first time and it took me a while to realize that it's the `$<` concatenated with a string and not some weird make syntax.

What about something like this:
``` make
cp --recursive "$(abspath $</signing_service/)" "$@"
```

This may add an extra `/` but `$(abspath)` will remove it.

Anyway, if it's a problem then forget about it. Either way it looks weird at first.

Reply:
Done.